docs/ManualDeploymentGuideScalarDLonAWS.md

```diff
@@ -0,0 +1,292 @@
+> * `helm ls -a` command can be used to list currently installed releases.
+> * You should confirm the load-schema deployment has been completed with `kubectl get po -o wide` before installing Scalar DL.
+> * Follow the [Advanced Scalar DL Configuration](#advanced-scalar-dl-configuration) section for more customization.
+
+## Step 5. Monitor the Cluster
+
+It is critical to actively monitor the overall health and performance of a cluster running in production. 
+You can use container insights to collect performance metrics and Fluent Bit to collect logs of the EKS cluster. 
+This section will help you to configure monitoring and logging for your EKS cluster.
+
+### Recommendations
+
+* Monitoring should be enabled for EKS in production.
+* CloudWatch agent should be configured in the EKS cluster for collecting metrics from pods.
+* Fluent Bit should be configured in the EKS cluster for collecting logs from pods.
+    * Node instance role must have `CreateLogGroup` permission for Fluent bit.
+
+### Setup container insights
+
+CloudWatch Container Insights helps you to collect, aggregate, and summarize metrics and logs from your containerized applications, so you can set up a cloudwatch agent in the node group created for log and metrics collection.
+
+[Setup cloudwatch agent to collect cluster metrics](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Container-Insights-setup-metrics.html) will help you to collect metrics from your containers.
+
+### Setup the Fluent Bit
+
+Fluent Bit allows you to collect any data like metrics and logs from different sources, so you can set up the Fluent Bit in the node group created for log and metrics collection.
+
+[Setting up Fluent Bit](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Container-Insights-setup-logs-FluentBit.html#Container-Insights-FluentBit-setup)  will help you to collect logs from containers.
+
+## Step 6. Checklist for confirming Scalar DL deployments
+
+After the Scalar DL deployment, you need to confirm that deployment has been completed successfully. This section will help you to confirm the deployment.
+
+### Confirm Scalar DL deployment
+
+You can check if the pods and the services are properly deployed by running the `kubectl get po,svc,endpoints -o wide` command on the bastion host
+
+* Make sure the load balancer's private endpoint (EXTERNAL-IP) is created properly.
+* Make sure the ledger and envoy pods are running properly.
+* Make sure the schema loading pod execution is completed properly.
+* Make sure the schema is properly created in the underlying database service.
+
+### Confirm EKS cluster monitoring
+
+* Confirm the EKS cluster metrics are available in CloudWatch `Container Insights`.
+* Confirm the Container logs are available in CloudWatch `Log`.
+
+### Confirm Database monitoring
+
+* Confirm the database metrics are available in CloudWatch metrics.
+
+## Advanced Scalar DL configuration
```

Comment:
As I said before, I don't think it is advanced. See the cockroach DB doc. (like I said several times...)

Reply:
Updated the heading to Maintain the cluster.